src/App.js

```diff
@@ -99,12 +99,12 @@ class App extends React.Component {
       return
     }
     // For providers supporting .enable() (EIP 1102 draft).
-    if ('enable' in provider) {
+    if (typeof provider.enable === 'function') {
       provider.enable()
       return
     }
     // For providers supporting EIP 1102 (final).
-    if ('send' in provider) {
+    if (typeof provider.send === 'function') {
```

Comment:
Just as a note, the idea of using `in` was to be as paranoid as possible with this object: prevent triggering a getter by reading a property, or a type error if the injected `provider` is, for any reason, not an object. But these precautions are probably unnecessary and only in some places in the codebase, so let’s do it like this 👍